### ISSUE TITLE
Reversed coordinate order in get_product_info() ?

### ISSUE THREAD

**Issue by JohnLM** [ibamacsr/sentinelsat#34]
The [piece of code in `SentinelAPI.get_product_info()`](https://github.com/ibamacsr/sentinelsat/blob/83a1393447e5ab00057a915e1b61425d5da5f03e/sentinelsat/sentinel.py#L215-L224)  which formats the footprint string has a misleading comment. Specifically it says:
> ```
        # parse the GML footprint to same format as returned
        # by .get_coordinates()
```

which is not true as GML -- parsed by the function -- stores coordinates in (lat, lon) order, while the GeoJSON -- used by `get_coordinates()` -- stores them in (lon, lat) order.
Thus the coordinates are actually returned in reverse coordinate order than the `get_coordinates()` format.

I'm not sure how much code is out there which relies on functionality of `SentinelAPI.get_product_info()`, but fixing it may break such code. If that is a concern, I'd just modify the comment.

**Comment by Fernerkundung** [comment]
Great catch @JohnLM!

I fixed this in #49 - all returned coordinate pairs are now in lon,lat order.

**Comment by Fernerkundung** [comment]
This has been solved with #49 and can be closed @JohnLM @willemarcel
